modules/op-creating-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -63,19 +64,44 @@ You can use the following example to match the `pipeline-push-on-main` pipeline
 
 [source,yaml]
 ----
-...
-  metadata:
-    name: pipeline-push-on-main
-  annotations:
-    pipelinesascode.tekton.dev/on-target-branch: "[refs/heads/main]" <1>
-    pipelinesascode.tekton.dev/on-event: "[push]"
-...
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: pipeline-push-on-main
+annotations:
+  pipelinesascode.tekton.dev/on-target-branch: "[refs/heads/main]" <1>
+  pipelinesascode.tekton.dev/on-event: "[push]"
+# ...
 ----
-<1> You can specifiy multiple branches by adding comma-separated entries. For example, `"[main, release-nightly]"`. In addition, you can specify the following:
+<1> You can specify multiple branches by adding comma-separated entries. For example, `"[main, release-nightly]"`. In addition, you can specify the following:
 * Full references to branches such as `"refs/heads/main"`
 * Globs with pattern matching such as `"refs/heads/\*"`
 * Tags such as `"refs/tags/1.\*"`
 
+[discrete]
+.Matching a comment event to a pipeline run
+
+You can use the following example to match the `pipeline-comment` pipeline with a comment on a pull request, when the text of the comment matches the `^/merge-pr` regular expression:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: pipeline-comment
+annotations:
+  pipelinesascode.tekton.dev/on-comment: "^/merge-pr"
+# ...
+----
+
+The pipeline run starts only if the comment author meets one of the following requirements:
```

Comment:
we create a PipelineRun to start a Pipeline in general in tekton and Pac only really deals with pipelinerun so you may want to change that..

Reply:
I've made the changes I believe